### ISSUE TITLE
ino-nav-drawer: Enable navigation via keyboard

### ISSUE THREAD

**Issue by silentHoo** [inovex/elements#324]
Operation with the keyboard should be possible. The arrow keys up and down are to be used for this. The autofocus should not be affected.

**Acceptance criteria**
* Keyboard navigation with arrow up/down should be possible if a "tab" has the focus.
* Functionality analogous to the ino-tab-bar. Selection with enter or space key.
